### PR TITLE
MAT-3812 Test case to Retrieve Measures created by LoggedIn Users only and MAT-3753 Test case to add JSON Editor for Test Cases.

### PR DESCRIPTION
**Pull request by praveenavas**
MAT-3812 Test case to Retrieve Measures created by LoggedIn Users only and MAT-3753 Test case to add JSON Editor for Test Cases.

### CODE REVIEW

File: cypress/integration/Services/Measure Service/Measure.spec.ts [hunk 1]
```diff
@@ -40,6 +40,46 @@ describe('Measure Service: Create Measure', () => {
...
+
+        cy.getCookie('accessToken').then((accessToken) => {
+            cy.request({
+                url: '/api/measures?currentUser=true',
+                method: 'GET',
+                headers: {
+                    authorization: 'Bearer ' + accessToken.value
+                }
+            }).then((response) => {
+                expect(response.status).to.eql(200)
```
Comment (cvasile001) [inline cypress/integration/Services/Measure Service/Measure.spec.ts:74]: are we also checking the array length verifying that?

**Review by cvasile001 — COMMENTED** [review]
Left one comment

File: cypress/integration/Services/Measure Service/Measure.spec.ts (hunk 1 above)
Comment (praveenavas) [reply cypress/integration/Services/Measure Service/Measure.spec.ts:74]: Yes this line ' cy.get(response.body.length)' will give the length of the array

**Review by cvasile001 — APPROVED** [review]
All good

**Review by cvasile001 — APPROVED** [review]
all good

**Pull request merged**